lib/build.js

```diff
@@ -344,19 +316,15 @@ function build(params) {
           Runtime: 'nodejs6.10',
           Timeout: 30
         }
-      },
-      SpotFleet: {
-        Name: params.LogicalName,
-        Type: 'Custom::SpotFleetFunction',
-        Properties: {
-          ServiceToken: cf.getAtt('SpotFleetFunction', 'Arn'),
-          SpotFleetRequestConfigData: params.Properties.SpotFleetRequestConfigData,
-          Region: params.Properties.SpotFleetRegion
-        }
       }
     }
   };
 
+  customFunctions[params.CustomFunctionName][params.LogicalName] = {
+    Type: cf.join(['Custom::', params.CustomFunctionName, 'Function']),
```

Comment:
This part isn't actually important that you adjust. They could all be `Custom::Bananas` if you want -- they just have to start with `Custom::` and there's no fear of overlap.